modules/dataquery/jsx/react.tabs.js

```diff
@@ -118,6 +121,7 @@ ViewDataTabPane = React.createClass({
                 link.download = "data-" + dataDate + ".csv";
                 link.type = "text/csv";
                 link.href = dataURL;
+                document.body.appendChild(link);
```

Comment:
This was previous intentionally omitted because it's just a hack to start the download and not intended to be a real link. Does it break in Firefox if it's not appended? If so, it should also have display: none set in the CSS, and if possible be removed from the DOM after being clicked.